component/webui/src/main/java/org/exoplatform/social/webui/activity/BaseUIActivity.java

```diff
@@ -577,8 +578,9 @@ public boolean isActivityDeletable() throws SpaceException {
       }
     }
 
+    UserACL userACL = getApplicationComponent(UserACL.class);
     if (space != null) {
-      return spaceService.isManager(space, Utils.getOwnerRemoteId());
+      return userACL.isSuperUser() || spaceService.isSuperManager(Utils.getOwnerRemoteId()) || spaceService.isManager(space, Utils.getOwnerRemoteId()) || ConversationState.getCurrent().getIdentity().getGroups().contains(userACL.getAdminGroups());
```

Comment:
The check on platform administrators group is no more needed as we check on Space administration group

Reply:
Done